### PR TITLE
add dependabot config

### PR DESCRIPTION
**Pull request by Nexucis**

### CODE REVIEW

File: .github/dependabot.yml [hunk 1]
```diff
@@ -0,0 +1,17 @@
+# Until bug below is sorted we will not allow dependabot to run by itself for npm dependencies
+# https://github.com/dependabot/dependabot-core/issues/1190
```
Comment (ibakshay) [inline .github/dependabot.yml:2]: Do we need this comment for this repository? :) 
```suggestion
```

Comment (Nexucis) [reply .github/dependabot.yml:2]: ah yeah indeed not needed here 😅

Comment (Nexucis) [reply .github/dependabot.yml]: thanks :)

**Review by ibakshay — APPROVED** [review]
Thanks a lot! LGTM 🚀

**Pull request merged**